### PR TITLE
'show-credential' for local credentials.

### PR DESCRIPTION
**Pull request by anastasiamac**
## Description of change

'show-credential' displayed details of the credentials stored remotely on the controller. This PR adjust the command to display details of locally stored credentials too. 

## QA steps

* 'juju show-credentials' shows both locally and remotely stored credentials (use -c to display credentials from controller other than current) for logged on user;
* 'juju-show-credentials --local' shows only local credentials
* 'juju show-credentials <cloud name> <credential name>' shows details of local and remote credential with name <credential name> for cloud with name <cloud name>. 

## Bug reference

https://bugs.launchpad.net/juju/+bug/1786862
and part of solution for  https://bugs.launchpad.net/juju/+bug/1814395

### CODE REVIEW

**Comment by anastasiamac** [comment]
[show-credentials for local credentials](https://trello.com/c/7kEzDd5V/848-show-credentials-for-local-credentials)

**Comment by hpidcock** [comment]
Is there any documentation needing to be updated here https://docs.jujucharms.com/credentials ?

File: cmd/juju/cloud/showcredential.go [hunk 1]
```diff
@@ -49,6 +51,7 @@ func (c *showCredentialCommand) SetFlags(f *gnuflag.FlagSet) {
 		"yaml": cmd.FormatYaml,
 	})
 	f.BoolVar(&c.ShowSecrets, "show-secrets", false, "Display credential secret attributes")
+	f.BoolVar(&c.Local, "local", false, "Local operation only; controller not affected")
```
Comment (hpidcock) [inline cmd/juju/cloud/showcredential.go]: Show credentials shouldn't "affect" the controller. I'd reword this to something like "Local operation only; controller credentials not listed"

**Comment by anastasiamac** [comment]
@hpidcock,
No, there is no mentioning of this command in the referenced documentation page.

File: cmd/juju/cloud/showcredential.go (hunk 1 above)
Comment (anastasiamac) [reply cmd/juju/cloud/showcredential.go]: This is a standard description for all --local options.

Comment (babbageclunk) [reply cmd/juju/cloud/showcredential.go]: Right, but in most cases the command does something on/to the controller - here it's confusing. It's more important for it to make sense in the context of this command than to be consistent globally (when they pull in different directions).

**Comment by anastasiamac** [comment]
$$merge$$
